aws-observability-terraform/source-module/update_sources.tf

```diff
@@ -64,6 +64,28 @@ resource "null_resource" "AddFieldsToELBSource" {
    }
 }
 
+# Add fields to source if Classic LoadBalancer source already exists
+resource "null_resource" "AddFieldsToCLBSource" {
+   for_each   = toset(local.update_classic_lb_source ? ["add_fields_to_source"] : [])
+   triggers = {
+       access_id         = var.access_id
+       access_key        = var.access_key
+       env               = var.environment
+       source_url        = var.elb_log_source_url
```

Comment:
I presume this should of classic lb - classic_lb_log_source_url

Reply:
Done